src/fontra/views/editor/panel-transformation.js

```diff
@@ -580,16 +580,17 @@ export default class TransformationPanel extends Panel {
     await this.sceneController.editGlyphAndRecordChanges(
       (glyph) => {
         for (const [layerName, layerPath] of Object.entries(layerPaths)) {
+          const newPackedPath = VarPackedPath.fromObject(layerPath);
```

Comment:
Suggest to do the conversion in server-utils.js.

Reply:
(In retrospect, parseClipboard() should do the same, there's now a conversion method in editor.js. For later maybe.)